lib/cs_guide/categories/drink_type.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CsGuide.Categories.DrinkType do
+  use Ecto.Schema
+  use Alog
+  import Ecto.Changeset
+
+  schema "drink_types" do
+    field(:name, :string)
+    field(:entry_id, :string)
+    field(:deleted, :boolean)
```

Comment:
`alog` should automatically add the `:deleted` column to any table where it is used. 
see: https://github.com/dwyl/phoenix-ecto-append-only-log-example/issues/7#issuecomment-431863209